### PR TITLE
feat-support-additional-start-times

### PR DESCRIPTION
**Pull request by EdLeckert**
Feature to support additional start times (`start1`-`start3`) and not require clients to encode/decode bits for start times.

PR for `hass-opensprinkler` to follow.

No known breaking changes.

Plan to support `Repeating`start times shortly, which should complete this feature set.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
     author="Vincent Teo, Travis Glenn Hansen",
     author_email="vinteo@gmail.com, travisghansen@yahoo.com",
     packages=["pyopensprinkler"],
-    install_requires=["aiohttp==3.8.5", "backoff==2.2.1"],
+    install_requires=["aiohttp>=3.8.5", "backoff>=2.2.1"],
```
Comment (vinteo) [inline setup.py:21]: weird, how come this is part of the commits, confused me for a minute. Can you rebase and squash?

Comment (EdLeckert) [reply setup.py:21]: Probably because it was branched and I was working on it before the aiohttp change.

**Review by vinteo — APPROVED** [review]

**Comment by vinteo** [comment]
actually sorry to be a pain but a more meaningful commit message would be good

File: setup.py (hunk 1 above)
Comment (EdLeckert) [reply setup.py:21]: So, it's an even bigger mess now, but I think the commit message is good.

**Comment by vinteo** [comment]
Will do a release together with the repeating start times once that goes in

**Pull request merged**